### PR TITLE
Update reconnect_task.py

### PR DESCRIPTION
**Pull request by zhoushunping**
Domain name resolution should be performed again when reconnecting

### CODE REVIEW

**Comment by zhoushunping** [comment]
When the primary node is unavailable, it will automatically switch to the standby node according to the new address resolved by the domain name

File: chia/server/reconnect_task.py [hunk 1]
```diff
@@ -3,26 +3,31 @@
...
-    async def connection_check():
+    async def connection_check(peer_info: PeerInfo):
         while True:
             peer_retry = True
             for _, connection in server.all_connections.items():
                 if connection.get_peer_info() == peer_info or connection.get_peer_info() == peer_info_arg:
                     peer_retry = False
             if peer_retry:
+                if server._local_type == NodeType.HARVESTER:
+                    host = socket.gethostbyname(peer_info_arg.host)
```
Comment (arvidn) [inline chia/server/reconnect_task.py:23]: this is a blocking call, that may lock up all other tasks. This has to be async. We use an async DNS library in other parts of the code

**Review by arvidn — CHANGES_REQUESTED** [review]

**Review by zhoushunping — COMMENTED** [review]
We use an async DNS library for a non-blocking call

**Review by zhoushunping — COMMENTED** [review]
finished

**Review by duongxuantuan154 — APPROVED** [review]
- [ ]

**Comment by emlowe** [comment]
I believe this PR needs to have some substantial updates as the current code has moved. Closing PR. It can be re-opened if updated

**Pull request closed without merging**